core/templates/dev/head/components/social_buttons/SocialButtonsDirective.js

```diff
@@ -0,0 +1,29 @@
+// Copyright 2016 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for the social buttons displayed in footer
```

Comment:
Really sorry for this :P. today is actually weird for me. i am makin a commit and then forget that i have missed some other minor thing. Even before this i added full stop with bunch more text, but there was 80 character violation. Thanks for being patient with me.